package.json

```diff
@@ -19,15 +19,18 @@
     "@emotion/styled": "^11.8.1",
     "@mui/material": "^5.6.2",
     "@nrwl/next": "13.10.3",
+    "@radix-ui/react-accordion": "^0.1.6",
     "@radix-ui/react-dialog": "^0.1.5",
     "@radix-ui/react-dropdown-menu": "^0.1.6",
+    "@radix-ui/react-icons": "^1.1.1",
     "@radix-ui/react-radio-group": "^0.1.5",
     "@radix-ui/react-tabs": "^0.1.5",
     "@radix-ui/react-tooltip": "^0.1.7",
     "@sentry/nextjs": "^6.19.3",
     "@sentry/react": "^6.19.2",
     "@sentry/tracing": "^6.19.2",
-    "@vegaprotocol/vegawallet-service-api-client": "^0.4.11",
+    "@stitches/react": "^1.2.8",
```

Comment:
question (blocking): I'm not sure about adding this dep. It seems like a pretty full featured css in js solution when we already have tailwind. Wondering if there is a simpler solution which doesn't muddy the waters of how we style things